tokio/src/runtime/scheduler/multi_thread/handle.rs

```diff
@@ -28,6 +29,9 @@ pub(crate) struct Handle {
 
     /// Current random number generator seed
     pub(crate) seed_generator: RngSeedGenerator,
+
+    /// User-supplied hooks to invoke for things
+    pub(crate) scheduler_hooks: TaskHooks,
```

Comment:
We already have some hooks. Is there a reason to not put the new hooks in the same place?

Reply:
I figured it'd be neater to store the task-related hooks somewhere separate.

Reply:
Do you want this moved elsewhere?

Reply:
Maybe `task_hooks` is a better name, then?

Reply:
Will do

Reply:
Done